lineman/app/components/thread_page/thread_page.haml

```diff
@@ -1,52 +1,53 @@
-.loading-wrapper.container.main-container
+.loading-wrapper.lmo-column-container.main-container
   %loading{ng-if: '!threadPage.discussion'}
   %main.thread-page{ng-if: 'threadPage.discussion'}
     %group_theme{group: 'threadPage.discussion.group()', compact: 'true'}
 
-    %section.thread-context{aria-label: "{{ 'thread_context.aria_label' | translate }}"}
```

Comment:
This looks like a lot but it's just shifting the cards into divs with `.lmo-column-left` and `.lmo-column-right` classes